dump/dist/pvm_dump_offload.service.in

```diff
@@ -1,11 +1,11 @@
 [Unit]
-Description=PowerVM Handler
+Description=PowerVM Handler %i
 Wants=phosphor-debug-collector.service
+Wants=obmc-host-started@%i.target
 After=phosphor-debug-collector.service
 After=xyz.openbmc_project.biosconfig_manager.service
```

Comment:
what is the use case of adding biosconfig_manager service with new changes?

Reply:
it was there earlier too, I will be reading an attribute from biosconfig pvm_hmc_managed so I want to ensure that the service is running.